tests/e2e/tests/test_clustering.py

```diff
@@ -32,6 +33,7 @@ def add_node(h: harness.Harness, cluster_node: str, joining_node: str) -> str:
         ["k8s", "add-node", joining_node],
         capture_output=True,
     )
+    assert out.returncode == 0
```

Comment:
i think `h.exec` will raise an exception if  the returncode is non-zero.  There shouldn't be a need for this